modules/ROOT/pages/install-azure.adoc

```diff
@@ -12,8 +12,7 @@ Before installing Runtime Fabric ensure your infrastructure meets the minimum ha
 and networking requirements.
 
 Your organization's operations, networking and security teams should review these requirements. 
-See xref:install-sys-reqs.adoc[System Requirements for Anypoint Runtime Fabric] and 
-xref:install-port-reqs.adoc[Network and Port Requirements for Anypoint Runtime Fabric].
+See xref:install-prereqs.adoc[Anypoint Runtime Fabric Installation Prerequisites].
 ====
 
 == Before you Begin
```

Comment:
```suggestion
== Before You Begin
```
At some point, we will standardize between this and "Prerequisites" or other versions being used by various writers.